source/main.js

```diff
@@ -244,3 +244,19 @@ ipcMain.handle('update-markdown-entry', async (event, arg) => {
         fs.writeFileSync(entries[entryIndex].fileName, arg.markdownContent, 'utf-8');
     }
 });
+
```

Comment:
Make sure to add proper JSDoc documentation to these methods. Refer to the ones right above this to see examples